pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -256,13 +256,13 @@ func (e *AzureResourceGraphDatasource) unmarshalResponse(res *http.Response) (Az
 
 func getAzurePortalUrl(azureCloud string) (string, error) {
 	switch azureCloud {
-	case "azuremonitor":
+	case "AzureCloud":
```

Comment:
nit: there're constants in the setting package ([example](https://github.com/grafana/grafana/blob/f32d200fc0047810e03255d38cced0ed36f983ce/pkg/tsdb/azuremonitor/aztokenprovider/token_provider.go#L107))